### PR TITLE
Add german to the app! 🇩🇪

### LINKED ISSUE #1142: Support German lang
The work here consists in adding German in the language selection screen and updating the related config in https://github.com/valora-inc/wallet/blob/main/packages/mobile/locales/index.ts

The Xcode project also has to be updated so that the App Store recognizes German is supported:

Then only select the plist:

### PR DESCRIPTION
**Pull request by isabellewei**
### Description

German enabled in the app and appears in the language selection screen. Strings will need to be updated in Crowdin

### Related issues

- Fixes #1142

### CODE REVIEW

**Review by jeanregisser — COMMENTED** [review]
Thanks!

File: packages/mobile/locales/index.ts [hunk 1]
```diff
@@ -36,6 +36,15 @@ const locales: Locales = {
       return require('date-fns/locale/pt-BR')
     },
   },
+  de_DE: {
```
Comment (jeanregisser) [inline packages/mobile/locales/index.ts:39]:
```suggestion
  de-DE: {
```

File: packages/mobile/locales/de-DE/accountScreen10.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
```
Comment (jeanregisser) [inline packages/mobile/locales/de-DE/accountScreen10.json:1]: We should leave empty json files `{}` to avoid conflicts with Crowdin.

**Review by jeanregisser — CHANGES_REQUESTED** [review]

File: packages/mobile/locales/de-DE/accountScreen10.json (hunk 2 above)
Comment (jeanregisser) [reply packages/mobile/locales/de-DE/accountScreen10.json:1]: And rename the folder from `de-DE` to `de`.

Comment (gnardini) [reply packages/mobile/locales/de-DE/accountScreen10.json:1]: Crowin creates the files if they don't exist, so just deleting them works too

**Review by jeanregisser — APPROVED** [review]

**Pull request merged**